.github/workflows/shippable_builds.yml

```diff
@@ -566,20 +566,24 @@ jobs:
         run: |
           true
 
-  publish_release:
-    name: Publish Release
-    needs: [pre_publish, dump_config, release_commit]
+  pkginfo:
+    # Generate pkginfo used by the publish jobs and the release notes
+    name: Generate PkgInfo
+    needs: [publish_hold, dump_config, release_commit]
     if: ${{ !failure() && !cancelled() }} # Run if previous step is skipped
     runs-on: ubuntu-latest
     strategy:
       matrix:
         include: "${{ fromJSON(needs.dump_config.outputs.matrixInclude) }}"
-    environment: publish_release
     outputs:
-      thunderbird_release_url: ${{ steps.summary.outputs.thunderbird_release_url }}
-      k9mail_release_url: ${{ steps.summary.outputs.k9mail_release_url }}
-      thunderbird_full_version_name: ${{ steps.summary.outputs.thunderbird_full_version_name }}
-      k9mail_full_version_name: ${{ steps.summary.outputs.k9mail_full_version_name }}
+      TAG_NAME: ${{ steps.pkginfo.outputs.TAG_NAME }}
+      FULL_VERSION_NAME: ${{ steps.pkginfo.outputs.FULL_VERSION_NAME }}
+      VERSION_NAME: ${{ steps.pkginfo.outputs.VERSION_NAME }}
+      VERSION_CODE: ${{ steps.pkginfo.outputs.VERSION_CODE }}
+      APPLICATION_ID: ${{ steps.pkginfo.outputs.APPLICATION_ID }}
+      app_sha: ${{ steps.shanotes.outputs.app_sha }}
+      app_github_notes: ${{ steps.shanotes.outputs.app_github_notes }}
```

Comment:
Let's use a consistent format with outputs

Reply:
Also, if you are using a Matrix job, the outputs will overwrite each other, so you only get one random set of variables.

If we need all of these later, it might make sense that instead of picking them from the apk/aab, we generate them from the sources. We have most of this already in the appinfo step early on, we just need to put it into the right outputs.